validators/cgroup_validator.go

```diff
@@ -102,5 +138,8 @@ func (c *CgroupsValidator) getCgroupSubsystems() ([]string, error) {
 			}
 		}
 	}
+	if err := s.Err(); err != nil {
+		return nil, err
+	}
```

Comment:
could you explain why this is needed?
this is already handled on line 130.

Reply:
It's my mistake from the old commit. I removed this code.